packages/mui-system/src/createBox.js

```diff
@@ -11,7 +11,10 @@ export default function createBox(options = {}) {
     generateClassName,
     styleFunctionSx = defaultStyleFunctionSx,
   } = options;
-  const BoxRoot = styled('div')(styleFunctionSx);
+  const BoxRoot = styled('div', {
+    shouldForwardProp: (prop) =>
+      prop !== 'ownerState' && prop !== 'theme' && prop !== 'sx' && prop !== 'as',
```

Comment:
```suggestion
    shouldForwardProp: (prop) => prop !== 'sx',
```

This should be enough.

Reply:
@siriwatknp 
When I only use `prop !== 'sx'` on the system baisc page of the local documentation site to have a test, this seems not enough. I am not sure whether we need the `ownerState` there. 

```diff 
--- Demo.tsx
+++ Demo.tsx
@ -37,10 +37,10 @@ export default function Demo() {
          minWidth: { md: 350 },
        }}
      >
-        <Box component="span" sx={{ fontSize: 16, mt: 1 }}>
+        <Box component={Box} sx={{ fontSize: 16, mt: 1 }}>
          123 Main St, Phoenix AZ
        </Box>
-        <Box component="span" sx={{ color: 'primary.main', fontSize: 22 }}>
+        <Box component={Box} sx={{ color: 'primary.main', fontSize: 22 }}>
          $280,000 — $310,000
        </Box>
        <Box
```
![example](https://user-images.githubusercontent.com/85668115/182866770-b1068abb-07cc-4f26-a315-2593e961937c.PNG)

Reply:
Ah, you are right.